doc/configuration.rst

```diff
@@ -578,14 +578,15 @@ PAGURE_AUTH
 ~~~~~~~~~~~~
 
 This configuration key specifies which authentication method to use.
-Pagure currently supports three authentication methods: the first one
-is relying on the Fedora Account System
-`FAS <https://admin.fedoraproject.org/accounts>`_, the second is using
-OpenID Connect (any provider) and the third is using only the local database.
-It can therefore be either ``fas``, ``oidc`` or ``local``.
-
-If ``oidc`` is used, the configuration options starting with ``OIDC_``
-(see below) must be provided.
+Valid options are ``fas``, ``openid``, ``oidc``, or ``local``.
+
+* ``fas`` uses the Fedora Account System `FAS <https://admin.fedoraproject.org/accounts>` to provide user authentication.
+
+* ``openid`` uses the Fedora Account System without requring users to sign the Fedora Project Contributor Agreement.
```

Comment:
Note that you can use any provider as well, just specify the provider you want to use via: ``FAS_OPENID_ENDPOINT``